src/assets/styles/material-reduction.css

```diff
@@ -910,3 +910,34 @@ ul.nested-list{
 ul.nested-list li{
   margin-left: 24px;
 }
+
+.expanded-table {
```

Comment:
It would be better to keep styles localized to components when possible. Can this go to expandable-table.component.scss?

Reply:
The styles for material views don't get applied in localized files for some reason. That's why this file exists I think.